spec/models/employer_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+describe Employer do
+  let(:name) { "My Company" }
+  let(:account_number) { 1234567 }
+  let(:employer) { FactoryBot.build_stubbed(:employer) }
```

Comment:
If you've configured things correctly (I think you have in `rails_helper.rb`), you should be able to call `FactoryBot` methods directly.

```ruby
let(:employer) { build_stubbed(:employer) }
```

Reply:
Yes, that sound's correct, I'll try it